src/build.ts

```diff
@@ -97,9 +97,13 @@ export async function writeTypes(nitro: Nitro) {
   }
 
   let autoImportedTypes: string[] = [];
+  let autoImportExports: string;
 
   if (nitro.unimport) {
     await nitro.unimport.init();
+    autoImportExports = await nitro.unimport
+      .toExports(typesDir)
+      .then((r) => r.replace(/#internal\/nitro/g, runtimeDir));
```

Comment:
There must be a better solution for this 🤔

Reply:
Haha just added same comment. What is issue?

Reply:
The issue is that `#internal/nitro` type isn't exposed to users by default - the types of all of these imports will be `any` unless we resolve it